NOTES.md

```diff
@@ -1,3 +1,186 @@
+# Release 2.0.0-RC1 (Musashi)
```

Comment:
Can we leave the release name off, as this is an RC release? (presumably it should be consistent with the actual 2.0 release name).